js/defaults.js

```diff
@@ -8,7 +8,7 @@
  */
 
 var defaults = {
-	port: 8080,
+	port: process.env.MM_PORT || 8080,
```

Comment:
After MM_PORT there a stranger space. Irregular whitespace. In character 30. Add a normal space.